source/common/http/utility.h

```diff
@@ -621,6 +622,13 @@ AuthorityAttributes parseAuthority(absl::string_view host);
 envoy::config::route::v3::RetryPolicy
 convertCoreToRouteRetryPolicy(const envoy::config::core::v3::RetryPolicy& retry_policy,
                               const std::string& retry_on);
+
+/**
+ * @param request_headers the request header to be looked into.
+ * @return true if the request method is 0-RTT safe.
+ */
+bool isSafeRequest(Http::RequestHeaderMap& request_headers);
```

Comment:
nit: can you make this more informative for what it is? Potentially isZeroRttSafeRequest()? IsIdempotentRequest()? Something like that?

Reply:
I think "Safe" is the correct term to use, based on RFC 7231. (Amusingly, "Safe" is distinct in this context from "Idempotent" as the latter also includes PUT). Perhaps we could include the RFC link in this comment instead of in the .cc file?